picker-starter/README.md

```diff
@@ -22,7 +22,7 @@ As you can see in the example below, the default `platform` example function ret
 
 ```js
 export const platform: PickerPlatform<OptionsParams> = (options) => ({
-  itemServices: [
+  tabs: [
```

Comment:
I changed it from `itemServices` to `tabs` since, in this context, I think it's easier to understand 'tabs'.